### DISCUSSION TITLE
Alignment Vocabulary

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#63] [Q&A] [answered]
Hi,

As I was working through the reading I noticed that I didn't fully understand what certain terms meant for alignment especially in the samtools section (e.g, "proper pair" , "paired end bit" , basically any of the meanings in table 11-1). Its hard to conceptualize the reading content when I don't fully understand the context/definitions of certain things. 

So... I was wondering if anyone had some sort of glossary (the one at the end of the book didn't help) or other resource they could point me in the direction of that could help with deepening my understanding of these alignment-specific terms

Thanks in advance :)

**Comment by kubu4** [comment]
This is an excellent point!

This made me realize that we haven't covered the _source_ of all of this bioinformatics data: SEQUENCING!

@sr320 - Maybe you can give a brief overview of how sequencing technologies work (or, provide links to videos/diagrams)? I think that would help with some of the lingo being encountered.

**Reply by sr320** [reply]
See https://github.com/sr320/course-fish546-2021/discussions/65

**Comment by sr320** [comment, marked as answer]
These terms are based on the idea of paired-end sequencing is being used.

Here is a detailed explanation of this 
https://www.youtube.com/watch?v=WneZp3fSJIk

It might be good to watch the general video in #65 first.
